sentry-android-integration-tests/sentry-uitest-android/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ android {
     namespace = "io.sentry.uitest.android"
 
     defaultConfig {
-        minSdk = Config.Android.minSdkVersionNdk
+        minSdk = 21
```

Comment:
I think we also have `Config.Android.minSdkVersionCompose` for that, perhaps we could use it here